storage/innobase/os/os0file.cc

```diff
@@ -7884,11 +7885,15 @@ void os_aio_print_pending_io(FILE *file) { AIO::print_to_file(file); }
 
 #ifndef _WIN32
 void os_file_set_umask(mode_t umask) {
-  static bool was_already_set{false};
-  ut_a(!was_already_set);
-  was_already_set = true;
+  ut_a(!umask_was_already_set);
+  umask_was_already_set = true;
   os_innodb_umask = umask;
 }
+
+#ifdef XTRABACKUP
+void os_file_allow_reset_umask() { umask_was_already_set = false; }
```

Comment:
and set the os_innodb_umask to default / some _MAX value in this reset.

Reply:
thanks Satya, done in the latest commit